packages/xo-web/src/common/sorted-table/index.css

```diff
@@ -15,3 +15,18 @@
   outline: 2px solid #366e98;
   outline-offset: -2px;
 }
+
+/* Menu item variant */
+.danger{
+    background-color: #990822;
+}
+.primary{
+  background-color: #366e98;
+}
+.success{
+  background-color: #28a745;
+}
+.warning{
+  background-color: #eca649;
+}
+
```

Comment:
Good try but the result is ugly.

Proposal:
- `primary`: bold
- `success`: bold green
- `warning`: bold orange
- `danger`: bold red

Reply:
@julien-f It Look better but, IMO red is too flashy.